plugins/example/example.coffee

```diff
@@ -0,0 +1,22 @@
+(() ->
+  class HelloWorldPlugin
+    @metadata =
+      name: "Hello World"
+      author: "Zachary Vance"
+      description: "Prints 'Hello World' when the plugin is loaded"
+      version: 1
+      stores_data: false
+      #data_version: 0
+      requires: []
```

Comment:
plugins.coffee seems to suggest this is supposed to be `requirements` (which I like better)